applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -199,6 +199,11 @@ def RunSolutionLoop(self):
             self.OutputSolutionStep()
 
 
+class GeomechanicsAnalysis(GeoMechanicsAnalysis):
```

Comment:
We like the suggestion. Our application needs to be updated to the Kratos formatting.

Could you update this PR by creating a new python file with the correct naming `geo_mechanics_analysis.py` and move this wrapper class to it. This should satisfy the ROM application. While providing us with an easier transition to the new expected naming scheme. Do you agree?

Reply:
After discussing a short internal discussion, we think the creation of a new script `geo_mechanics_analysis.py ` will not be required, because from our side we can come up with a more robust solution for loading the analysis stage class of the physics. 

It seems to us that the naming of the file as `geomechanics_analysis.py` cannot be considered an innapropriate name. For now, the addition of this dummy class will allow us to effortlessly move forward, and eventually it can be destroyed equally easily, as no other scripts will depend on it.

Regarding PR #11322 in which one function is splitted into two:

> CreateSolver(model, custom_settings) ==> CreateSolver(model, custom_settings) + CreateSolverByParameters(model, custom_settings, parallelism) 

this was done to keep the backward compatibility, and is present on all apps. 

We could arrange to have a discussion about the extend to which the format of the files is fixed, and how we are envisioning to evolve it in the future, which should help you in getting/staying close to the Kratos formatting. @roigcarlo @RiccardoRossi